example/lib/pages/network_tile_provider.dart

```diff
@@ -56,7 +56,7 @@ class NetworkTileProviderPage extends StatelessWidget {
               padding: EdgeInsets.only(top: 8.0, bottom: 8.0),
               child: Wrap(children: [
                 Text('This is a map that is using NetworkTileProvider.'),
-                Text('Turn on flight mode to see that images are cached.'),
+                Text('Turn on flight mode to see that images are not cached.'),
```

Comment:
I am unsure if this is the right way. If there is no caching, why should we tell the user about it? I would still suggest removing that, but let's @kengu or @johnpryan decide about that

Reply:
Yeah there's a whole shenanigans about caching. In the end I think it was decided that caching would not be mentioned anywhere apart from a dedicated caching section in the docs because the main provider doesn't cache properly and we didn't want things to be too confusing. That's what [this PR](https://github.com/fleaflet/flutter_map/pull/913) is for.
So I think it should be removed all together.

Reply:
@JaffaKetchup Okay, then in that case I propose also removing [that reference](https://github.com/fleaflet/flutter_map/blob/issues/829-nullsafety/lib/src/layer/tile_layer.dart#L116) that @andreandersson spotted

Reply:
I haven't checked that far into the NetworkTileProvider, so I don't know at all what it does. What I do know is that images are not cached, and that _might be interesting to developers?_ If anyone has a good description I'm very open to changing it...

But I'll most likely not take the time in the near future to look deeper into it and find a better description though.

Reply:
@andreandersson No problem! :)
I would propose something like

```
This Provider does not provide caching.\n
For further options about that, check flutter_map's README on GitHub.
```

Reply:
> @andreandersson No problem! :)
> I would propose something like
> 
> ```
> This Provider does not provide caching.\n
> For further options about that, check flutter_map's README on GitHub.
> ```

That seems a good solution when the PR mentioned above is merged

Reply:
I've assumed #913 will be merged as well, and updated the description here and in tile_layer.dart ;)